javascript/javascript2/week2/movies.js

```diff
@@ -44630,13 +44630,11 @@ console.log(moviesWithKeywords.length);
 
```

Comment:
you could alternatively use spread operator to add tags to movie objects.(for exercise  4)

Reply:
@Vahablotfi It's nice if you share some good reference for **spread operator** as I would like to learn it too